index.html

```diff
@@ -3410,6 +3410,191 @@ <h3>
         <h2>
 DID URL Dereferencing
         </h2>
+        <p>
+The DID URL dereferencing function dereferences a <a>DID URL</a> into
+a resource with contents depending on the <a>DID URL</a>'s components,
+including the <a>DID method</a>, method-specific identifier, query parameters,
+and fragment. When a <a>DID URL</a> is also a <a>DID</a>, this process
+ performs <a>DID Resolution</a>. The details of how this
```

Comment:
```suggestion
and fragment. This process depends on <a>DID resolution</a>
of the <a>DID</a> contained in the <a>DID URL</a>.
The details of how this
```

Event though technically I agree with this, it's a bit too concrete for my taste here, remember we wanted to only do abstract function definitions here, not describe the concrete algorithm behind them.